source/script2.cpp

```diff
@@ -6086,6 +6086,14 @@ INT_PTR CALLBACK InputBoxProc(HWND hWndDlg, UINT uMsg, WPARAM wParam, LPARAM lPa
 		if (hControl = GetDlgItem(hWndDlg, IDC_INPUTPROMPT))
 			SetWindowText(hControl, CURR_INPUTBOX.text);
 
+		// Use the system's current language for the button names:
+		typedef LPCWSTR(WINAPI*pfnUser)(int);
+		HMODULE hMod = LoadLibrary(L"user32.dll");
+		pfnUser mbString = (pfnUser)GetProcAddress(hMod, "MB_GetString");
```

Comment:
This function is documented: https://docs.microsoft.com/en-us/windows/win32/dlgbox/mb-getstring

Reply:
👍 Sorry about that then, although it doesn't specify which versions are supported. In any case, I think you should would verify the proc address and supply a default if needed. Also, you could use `GetModuleHandle` instead of `LoadLibrary` and skip the call to `FreeLibrary`, since __user32.dll__ is loaded by default.

Cheers.

Reply:
Yes, that makes sense. I will optimize the code.

Reply:
I don't think OS support is an issue.
http://undoc.airesoft.co.uk/user32.dll/MB_GetString.php